include/swift/Runtime/Concurrent.h

```diff
@@ -297,7 +297,7 @@ template <class EntryTy> class ConcurrentMap {
 
       // Try to set the edge to the new node.
       if (std::atomic_compare_exchange_strong_explicit(edge, &node, newNode,
-                                                  std::memory_order_release,
+                                                  std::memory_order_acq_rel,
```

Comment:
Okay, but I'm going to link to [C++ DR 2445](http://cplusplus.github.io/LWG/lwg-active.html#2445) in protest. :)